src/utils/changelog.ts

```diff
@@ -416,18 +421,23 @@ async function getMilestoneInfo(
     .join('\n');
 
   const { repo, owner } = await getGlobalGithubConfig();
-  const milestoneInfo = ((await getGitHubClient().graphql(`{
-    repository(name: "${repo}", owner: "${owner}") {
-      ${milestoneQuery}
+  const graphqlQuery = `{
+      repository(name: "${repo}", owner: "${owner}") {
+        ${milestoneQuery}
+      }
     }
-  }
 
-  fragment MilestoneFragment on Milestone {
-    title
-    description
-    state
-  }
-`)) as MilestonesDetailsResult).repository;
+    fragment MilestoneFragment on Milestone {
+      title
+      description
+      state
+    }
+  `;
+  logger.trace('Running graphql query:', graphqlQuery);
+  const milestoneInfo = ((await getGitHubClient().graphql(
+    graphqlQuery
+  )) as MilestonesDetailsResult).repository;
+  logger.trace('Query result:', milestoneInfo);
```

Comment:
The rest of the changes in this file are just adding `logger.trace` calls for GraphQL queries, correct? Anything substantive beyond that?

Reply:
Nothing. These were added to make debugging the issue easier and I then realized these should exist anyway and left them there.

Reply:
Sounds good, just wanted to make sure I wasn't missing something. :)